docs/build/build-guides-template-basic.md

```diff
@@ -120,79 +175,27 @@ Feel free to copy the configuration below and use it to ensure everything is in
 deployment. This function should replace the `local_testnet_config` function within
 `node/src/chain_spec.rs`:
 
-> **:warning: WARNING! :warning:** Make sure you replace the ParaId with the one you reserved
```

Comment:
Yesssss, happy to see all of this goes away now 😇

Reply:
There is still some cleanup to do on my end, I mostly need to back around to the PR to allow for an easier way to generate parachain specs - although, it's just really modifying a JSON after generating it ATM, which isn't terrible, so I may push it with that extra manual step.  WDYT?

Reply:
agree, I think editing a JSON file is well within the capabilities of anyone. It is not amazing devEx tho, but we can improve that next.